src/cats/interfaces/cat.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface Cat {
+  name: string
```

Comment:
@shogo-mitomo 
Similar observation for thr `cat` definition that is returned in response.
https://github.com/nestjs/nest/blob/016116a71c2f42be3d50eb310466bd67f6938d7c/sample/11-swagger/src/cats/classes/cat.class.ts#L3